src/Entity/Jecoute/Riposte.php

```diff
@@ -37,16 +41,22 @@
  *         "get": {
  *             "path": "/v3/ripostes/{id}",
  *             "requirements": {"id": "%pattern_uuid%"},
+ *             "normalization_context": {"groups": {"riposte_read"}},
+ *         },
+ *         "put": {
+ *             "path": "/v3/ripostes/{id}",
+ *             "requirements": {"id": "%pattern_uuid%"}
```

Comment:
`@security` annotation is missing here, no?

Reply:
No, all users with `ROLE_NATIONAL` can edit ripostes, same for deletion.
This `access_control` is sufficient https://github.com/EnMarche/en-marche.fr/pull/6131/files/d3e5a5a69dc799f9d117a1f1f61dbf5b11acd093#diff-369307f7dd7131da9161ccc215ba03c617be56a37d6d14c571714d2f13d0b4aaR27